lib/lookup.json

```
@@ -424,17 +349,8 @@
   },
   "rewire": {
     "prefix": "v",
-    "maintainers": "jhnns"
-  },
-  "rimraf": {
-    "prefix": "v",
-    "flaky": "win32",
-    "maintainers": "isaacs"
-  },
-  "router": {
-    "prefix": "v",
-    "maintainers": "dougwilson",
-    "skip": "win32"
+    "maintainers": "jhnns",
+    "scripts": ["test --timeout 5000"]
```

Shouldn't this be:

```suggestion
    "scripts": ["test -- --timeout 5000"]
```

? 😁

Running `npm run test --timeout 5000` throws `TypeError: arg.split is not a function`
`npm run test -- --timeout 5000` works.

---

That's what I get for not testing before committing. 🙃  (In my totally-not-convincing defense, I'm traveling and on limited connectivity so not doing things like cloning repos, etc. But, uh, still....)

---

Although given this result in CI, I guess it would running `npm run 'test -- --timeout 5000'` and that would still fail...

```
Missing script: "test --timeout 5000"
```

---

I'll update `lib/package-manager/lookup.js` so it handles this situation:

```
    "scripts": [["test", "--", "--timeout", "5000"]]
```

---

I may not want to generalize too much from a single run, but bumping up the timeout correlated with rewire passing everywhere in https://ci.nodejs.org/job/citgm-smoker/3239/testReport/. So that's good....